zivid_camera/src/settings_generator.cpp

```diff
@@ -248,6 +308,11 @@ class DynamicReconfigureCfgGenerator
       ss_ << ", " << valueTypeToRosTypeString(node.validRange().min()) << ", "
           << valueTypeToRosTypeString(node.validRange().max());
     }
+    else if constexpr (IsEnumSetting<SettingsNode>::value)
+    {
+      const auto max_index = node.validValues().size() - 1;
+      ss_ << ", 0, " << max_index << ", edit_method=" << rosGeneratedEnumVariableName(setting_name);
```

Comment:
Could you you name the 0 in ", 0," explicitly? I guess it's the min value for the enum, but that might be difficult to know for someone reading this code.
Also I guess we are assuming here that the min enum value is 0, and not 3 for instance. I think naming the zero would make the assumption clearer.

Reply:
Will fix

Reply:
done